db/migrate/20210617203927_migrate_admin_tax_amounts.rb

```diff
@@ -0,0 +1,52 @@
+class MigrateAdminTaxAmounts < ActiveRecord::Migration[6.0]
+  class Spree::Adjustment < ApplicationRecord
+    belongs_to :originator, polymorphic: true
+    belongs_to :adjustable, polymorphic: true
+    belongs_to :order, class_name: "Spree::Order"
+    belongs_to :tax_category, class_name: 'Spree::TaxCategory'
+    has_many :adjustments, as: :adjustable, dependent: :destroy
+
+    scope :admin, -> { where(originator_type: nil) }
+  end
+
+  def up
+    migrate_admin_taxes!
+  end
+
+  def migrate_admin_taxes!
+    Spree::Adjustment.admin.where('included_tax <> 0').find_each do |adjustment|
```

Comment:
Is it correct to assume that none of these adjustments have the tax adjustment already? For example, is all the current production code setting `included_tax` to `0`?

Reply:
"Admin" adjustments are the only remaining type (currently) that still make use of the `included_tax` attribute and don't have tax adjustments.

After this, we won't be using that database column any more for any adjustment types, all tax amounts will be recorded in tax adjustments. We can even drop that column from the table, but I thought I'd wait.

Reply:
So if we were super thorough, we would deploy the UI changes first and then run the migration so that nobody can set included tax while we are running it. That scenario seems very unlikely though.